easybuild/easyconfigs/u/UDUNITS/UDUNITS-2.2.19-intel-2015a.eb

```diff
@@ -0,0 +1,44 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2012-2013 University of Luxembourg, Ghent University
+# Authors::   Fotis Georgatos <fotis@cern.ch>, Kenneth Hoste (Ghent University)
+# License::   MIT/GPL
+# $Id$
+#
+# This work implements a part of the HPCBIOS project and is a component of the policy:
+# http://hpcbios.readthedocs.org/en/latest/HPCBIOS_2012-97.html
+##
+
+easyblock = 'ConfigureMake'
+
+name = 'UDUNITS'
+version = '2.2.19'
+
+homepage = 'http://www.unidata.ucar.edu/software/udunits/'
+description = """UDUNITS supports conversion of unit specifications between formatted and binary forms,
+ arithmetic manipulation of units, and conversion of values between compatible scales of measurement."""
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+toolchainopts = {'opt': True, 'pic': True}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ['ftp://ftp.unidata.ucar.edu/pub/udunits']
+
+dependencies = [('expat', '2.1.0')]
+
+sanity_check_paths = {
+    'files': [
+        'bin/udunits2',
+        'include/converter.h',
+        'include/udunits2.h',
+        'include/udunits.h',
+        'lib/libudunits2.a',
+        'lib/libudunits2.so',
```

Comment:
hmm, is this too long when collapsed on one line? looks a bit stretched out like this...

```python
    'files': ['bin/udunits2', 'include/converter.h', 'include/udunits2.h', 'include/udunits.h',
              'lib/libudunits2.a', 'lib/libudunits2.so'],
...
```